packages/appstore/src/components/modals/account-type-modal/account-type-modal.scss

```diff
@@ -0,0 +1,63 @@
+.account-type-wrapper {
+    display: flex;
+}
+
+.account-type--custom-exit {
+    display: none;
+}
+
+.account-type-card {
+    border: solid 1px var(--border-normal);
+    border-radius: 0.8rem;
```

Comment:
```suggestion
    border-radius: $BORDER_RADIUS * 2
```

Reply:
Done

Reply:
just out of curiosity, shouldn't we just define a new border radius instead of multiplying it by 2?
@mahdiyeh-fs @hirad-rewok 

I personally prefer to keep it as `0.8rem` because in case `BORDER_RADIUS` changed in future, this border will be double as well and it is not gonna be 0.8rem anymore

Reply:
I actually prefer the 0.8rem as well. I just made the change since everybody in reviews recommend using BORDER_RADIUS itself or sometimes multiplied.

Reply:
@hirad-rewok I suggest either reverting it back or define something like a BORDER_RADIUS_SM and use that one.

Reply:
![Uploading image.png…]()